### ISSUE TITLE
ML SK Tracker

### ISSUE THREAD

**Issue by phentos** [tritonhacks/Tritonhack2024-ML-Starter-Kit#21]
# Populate this issue with what is left to do

- [x] #3 
- [x] #22
- [x] #23
- [x] #26

**Comment by phentos** [comment]
@sunwoo604 reminder to populate what's needed 👍

**Comment by sunwoo604** [comment]
I think we are done with all of em but if you have any recommendations to put, please let us know. Also, if you want to view the instruction page, click the title of readme. Thank tou

**Comment by brianlu09** [comment]
@sunwoo604 

A couple notes to make the starter kit more like a starter kit:
- Clear outputs in the notebook and re-push to GH
- Selectively choose what code is given out, maybe have fill in the blanks instead
- More like a starter kit with more interactivity for students, less like an example project

**Comment by sunwoo604** [comment]
> @sunwoo604
> 
> A couple notes to make the starter kit more like a starter kit:
> 
> * Clear outputs in the notebook and re-push to GH
> * Selectively choose what code is given out, maybe have fill in the blanks instead
> * More like a starter kit with more interactivity for students, less like an example project

Yeah I am using a sample dataset to provide bunch of different methods that that should consider utilizing for their own project(though they can build on top of this) so it is not actually one flow but various different ideas
